packages/react/src/components/link-menu.module.d.css.ts

```
@@ -3,10 +3,32 @@ declare const styles = {
   'Button': '' as string,
+  'Icon': '' as string,
+  'PreviewBody': '' as string,
+  'Title': '' as string,
+  'Host': '' as string,
+  'Description': '' as string,
+  'ReplaceRow': '' as string,
+  'ReplaceButton': '' as string,
+  'SaveButton': '' as string,
+  'ReplaceButton': '' as string,
+  'Skeleton': '' as string,
+  'SkeletonLine': '' as string,
```

Biome (2.5.7)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)